.ruby-version

```diff
@@ -1 +1 @@
-2.7.6
+3.4.1
```

Comment:
I usually stay a little behind the latest ruby (especially in this case as 3.4 was only just released). I would suggest the latest of the `3.3.x` versions. For this particular gem I don't think it makes much difference.